base/common/wide_integer.h

```diff
@@ -127,7 +127,7 @@ class integer
     friend class std::numeric_limits<integer<Bits, signed>>;
     friend class std::numeric_limits<integer<Bits, unsigned>>;
 
-    base_type items[_impl::item_count];
+    base_type items[_impl::item_count] = {};
```

Comment:
No, we should not have it always initialized. It should behave in the same way as builtin types.

Reply:
Ok, seems that is a gcc bug.